deploy/helm/nfd-operator/values.yaml

```diff
@@ -3,20 +3,515 @@
 # Declare variables to be passed into your templates.
 namespace: node-feature-discovery-operator
 image:
-  repository: gcr.io/k8s-staging-nfd/node-feature-discovery-operator
+  repository: registry.k8s.io/nfd/node-feature-discovery-operator
   # This should be set to 'IfNotPresent' for released version
   pullPolicy: Always
   # tag, if defined will use the given image tag, else Chart.AppVersion will be used
-  # tag
+  # tag: devel
 imagePullSecrets: []
+labels: {}
+selectorLabels: {}
 
-nameOverride: ""
-fullnameOverride: ""
-namespaceOverride: ""
```

Comment:
If we delete these, then the templates/_helpers.tpl  should be adjusted accordingly.

These are basically for components that would have nfd-operator as a dependency (so that they would be able to install it in a separate ns etc.) Do we want to preserve this capability? If not, then we can remove these